### PR TITLE
fix(jobs): darkside proc box not working

### LINKED ISSUE #3087: jobs: After #3001 the drk Darkside proc box does not work
I haven't investigated this more than just filing the bug report.  The box does not trigger when I hit the Edge of Shadow action.  If I revert #3001, it starts working again.

### PR DESCRIPTION
**Pull request by MaikoTan**
Fixes #3087 

In fact, the bug likely occurred before #3001 though

### CODE REVIEW

File: webpack/webpack.config.cjs [hunk 1]
```diff
@@ -175,10 +175,6 @@ module.exports = function(env, argv) {
         },
       },
     },
-    cache: {
-      type: dev ? 'memory' : 'filesystem',
-      name: 'cactbot',
-    },
```
Comment (quisquous) [inline webpack/webpack.config.cjs:181]: This seems unrelated? If you wanted to remove this, can you send a different PR?

File: ui/jobs/components/drk.ts [hunk 2]
```diff
@@ -20,24 +20,24 @@ export const setup = (bars: Bars): void => {
...
+      } else {
+        p.classList.remove('low');
+        p.classList.remove('mid');
+      }
     }
 
     const oldSeconds = parseFloat(darksideBox.duration ?? '0') - parseFloat(darksideBox.elapsed);
     const seconds = jobDetail.darksideMilliseconds / 1000.0;
-    if (!darksideBox.duration ?? seconds > oldSeconds) {
+    if (!darksideBox.duration || seconds > oldSeconds) {
```
Comment (quisquous) [inline ui/jobs/components/drk.ts:40]: Whoops, I think this was a bad suggestion from me.  <_<

File: webpack/webpack.config.cjs (hunk 1 above)
Comment (MaikoTan) [reply webpack/webpack.config.cjs:181]: Oh yes, I'll remove that.

File: ui/jobs/components/drk.ts (hunk 2 above)
Comment (Trim21) [reply ui/jobs/components/drk.ts:40]: yes, blame to him (

Comment (Trim21) [reply ui/jobs/components/drk.ts:40]: I recall there are 3 or 4 suggestions like this

**Comment by MaikoTan** [comment]
Perhaps blame https://github.com/quisquous/cactbot/commit/8fc0673d32fd941aded29470cd781749509cb855

File: ui/jobs/components/drk.ts (hunk 2 above)
Comment (quisquous) [reply ui/jobs/components/drk.ts:40]: I double checked and I think the other `??` suggestions are ok.
